test/language/expressions/async-generators/early-errors-expression-NSPL-with-USD.js

```diff
@@ -0,0 +1,15 @@
+// Copyright 2017 the V8 project authors. All rights reserved.
+// This code is governed by the BSD license found in the LICENSE file.
+
+/*---
+author: Caitlin Potter <caitp@igalia.com>
+esid: pending
+description: >
+  It is a Syntax Error if ContainsUseStrict of AsyncGeneratorBody is true and
+  IsSimpleParameterList of UniqueFormalParameters is false.
+negative:
+  phase: early
+  type: SyntaxError
+---*/
+
+(async function*(x = 1) {"use strict"});
```

Comment:
this is already expected to throw an early error, see [line 10](https://github.com/tc39/test262/pull/821/files/26c33e6f4a3796434db48eea9b9b0ce38b85c0e0#diff-55c0af84ddfb413e1cfeedb1374b6134R10) for the negative flag expecting an early SyntaxError.